build-tools/xaprepare/xaprepare/xaprepare.targets

```diff
@@ -1,17 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <UsingTask TaskName="ReplaceFileContents" Condition=" '$(OS)' == 'Windows_NT' " TaskFactory="RoslynCodeTaskFactory" AssemblyFile="$(MSBuildToolsPath)\Microsoft.Build.Tasks.Core.dll">
+  <UsingTask TaskName="ReplaceFileContents" Condition=" '$(OS)' == 'Windows_NT' or '$(MSBuildRuntimeType)' != 'Mono' " TaskFactory="RoslynCodeTaskFactory" AssemblyFile="$(MSBuildToolsPath)\Microsoft.Build.Tasks.Core.dll">
     <ParameterGroup>
       <SourceFile ParameterType="Microsoft.Build.Framework.ITaskItem" Required="true" />
       <DestinationFile ParameterType="Microsoft.Build.Framework.ITaskItem" Required="true" />
       <Replacements ParameterType="System.String[]" />
       <ReplacementFilePath ParameterType="System.String" />
     </ParameterGroup>
     <Task>
-      <Code Type="Class" Language="cs" Source="..\..\xa-prep-tasks\Xamarin.Android.BuildTools.PrepTasks\ReplaceFileContents.cs"/>
+      <Code Type="Class" Language="cs" Source="../../xa-prep-tasks/Xamarin.Android.BuildTools.PrepTasks/ReplaceFileContents.cs"/>
```

Comment:
What's the story here?

Reply:
https://github.com/dotnet/msbuild/blob/bbb9655b007be6d079985f3a7ec14f5d82a18f64/src/Tasks/RoslynCodeTaskFactory/RoslynCodeTaskFactory.cs#L430

MSBuild uses `File.ReadAllText()` on that value, no "normalization" performed.